@shared/api/internal.ts

```diff
@@ -136,6 +136,7 @@ export const getAccountBalance = async (
     response = await server.loadAccount(publicKey);
   } catch (e) {
     console.error(e);
+    return { balance: "0" };
```

Comment:
this error doesn't appear to return a status code, so we can't explicitly check for 404 (which likely means it's a pubnet acct that hasn't been funded). Let's fail gracefully for now, but we can polish this experience a bit in the future

Reply:
Can we add TODO?